roles/preflight/tasks/test_preflight_check_container_one_image.yml

```diff
@@ -76,7 +76,7 @@
             {{ preflight_image }}
             check container
             {{ current_operator_image }}
-            {% if cert_project_id | default('') | length %}
+            {% if cert_project_id | default('') | length and dci_all_components_are_ga | default(True) %}
```

Comment:
Please consider renaming (with `preflight_` prefix) this var and documenting it

Reply:
Hi @tonyskapunk thanks for checking, I'm far from merging here. 
Somehow the OCP component on cluster10 where we have the latest GA [is not detected as a GA component](https://www.distributed-ci.io/jobs/a10685c6-442e-4f5f-83d0-9239390a05fb/jobStates?sort=date&task=bc33dc68-b1f0-477e-85d0-3aa0630b12b0):
```
task path: /home/dciteam/github/ansible-collection-redhatci-ocp-pr139-53ab370dfde4580c39d8e08339915c5f/dci-openshift-app-agent/plays/pre-run.yml:40
ok: [jumphost] => {
    "msg": [
        "dci_ga_components_for_certification: ['ocp']",
        "ga_components: []",
        "dci_all_components_are_ga: False"
    ]
}
```
Have to understand and fix that first.

Reply:
This component has `build:ga` tag but the query somehow does not pick it up
```
ga_components: "{{ job_info['job']['components'] | selectattr('tags', 'issuperset', ['build:ga']) | map(attribute='type') | list }}"
```
I will debug it tomorrow, I don't see the issue with the query right now :(

Reply:
can you display the components from the job?

Reply:
@tonyskapunk I added the changes you have requested. Could you please have a look one more time?